tests/test_cli/test_push.py

```diff
@@ -104,10 +104,9 @@ def setup_class(cls):
             standalone_mode=False,
         )
 
-    def test_exit_code_2(self):
```

Comment:
This test was wrong from beginning. The command was failing with exit code 2 but because of `--local` flag usage that was incorrect at that point. Normal behavior is exit with code 1, exception - SystemExit.